nucliadb_vectors/src/data_point_provider/mod.rs

```diff
@@ -129,16 +129,19 @@ impl Index {
         use std::collections::HashSet;
         let work_flag = self.work_flag.try_to_start_working()?;
         let state = self.read_state();
-        let non_garbage_dp: HashSet<_> = state.dpid_iter().map(|s| s.to_string()).collect();
+        let in_use_dp: HashSet<_> = state.dpid_iter().collect();
```

Comment:
`in_use_dp` stands for "data points in use"?

Reply:
yes